hazelcast-spring-tests/src/test/java/com/hazelcast/spring/CustomSpringJUnit4ClassRunner.java

```diff
@@ -81,4 +87,37 @@ public void evaluate() throws Throwable {
             }
         };
     }
+
+    /**
+     * includes {@link com.hazelcast.test.HazelcastTestSupport#smallInstanceConfig}
+     * except SqlConfig executor pool size, JetConfig cooperative thread count.
```

Comment:
Why don't we set the jet coop thread pool and the sql thread pools here as well?